pkg/utils/common.go

```diff
@@ -218,6 +247,22 @@ func NonRepeatStringsEqual(a, b []string) bool {
 	return true
 }
 
+// GenTableID generates table ID.
+func GenTableID(schema, table string) (id string, isSchemaOnly bool) {
+	if len(table) == 0 {
+		return fmt.Sprintf("`%s`", schema), true
```

Comment:
Will this have a better performance?
```suggestion
		return "`" + schema + "`", true
```